Exercises/Modules/Chapter 11/Soln11_05/Soln11_05.cpp

```diff
@@ -0,0 +1,25 @@
+// Splitting a module interface into partitions
```

Comment:
I can again not compile this in VC++, but code looks ok.

Reply:
Even after fixing the circular include, I could still not compile it...